data-collection/src/main/java/refactoringml/db/No.java

```diff
@@ -1,31 +1,30 @@
 package refactoringml.db;
 
 import refactoringml.util.RefactoringUtils;
-
 import javax.persistence.*;
-import java.util.Calendar;
 
+//TODO: create a Baseclass for both Yes and No, as they share a lot of logic
 @Entity
 @Table(name = "no", indexes = {@Index(columnList = "project_id"), @Index(columnList = "type"), @Index(columnList = "isTest")})
 public class No {
-
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private long id;
 
 	@ManyToOne
+	//project id: referencing the project information, e.g. name or gitUrl
 	private Project project;
 
-	private String commit;
-
-	@Temporal(TemporalType.TIMESTAMP)
-	private Calendar commitDate;
+	@ManyToOne(targetEntity = CommitMetaData.class)
```

Comment:
no need for targetEntity here, only if you are using interfaces/inheritance. Only the `@ManyToOne` should be enough.